state-chain/runtime/src/lib.rs

```diff
@@ -169,7 +169,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("chainflip-node"),
 	impl_name: create_runtime_str!("chainflip-node"),
 	authoring_version: 1,
-	spec_version: 140,
+	spec_version: 141,
```

Comment:
why the bump here? 131 is the last released runtime version. should be 140 here

Reply:
Didn't mean to commit this...